### ISSUE TITLE
Error when compiling templates.

### ISSUE THREAD

**Issue by rikschennink** [mjmlio/gulp-mjml#11]
Suddenly after reinstalling npm modules we get this error:
```
TypeError: must start with number, buffer, array or string`
```

We traced it to this line:
```js
output.contents = new Buffer(mjmlEngine.mjml2html(file.contents.toString()));
```

Changing it to this makes it work:
```js
output.contents = new Buffer(JSON.stringify(mjmlEngine.mjml2html(file.contents.toString())));
```

**Comment by iRyusa** [comment]
Hello there,

Can you check you're using MJML3 ? In MJML3, `mjml2html` returns an `Object` with `errors` and `html` properties.

**Comment by rikschennink** [comment]
The gulp-mjml package has mjml version set to "latest" so I guess it pulled 3.0.0 when I ran a fresh `nmp install` this afternoon.

**Comment by chibani** [comment]
Same error here. I tried to plug mjml with gulp.
I globally installed mjml 2.3.3 (because the 3.x doc is not available), unfortunately, gulp-mjml installs a 3.x version (aka "latest") :/

**Comment by Arcandias** [comment]
Same issue, I get exactly the same symptoms and I can't fix it because gulp-mjml using the latest version of mjml anyway.

Change the line to this also makes it work :
``js
output.contents = new Buffer(mjmlEngine.mjml2html(file.contents.toString()).html);
``

But I'd like to know if somebody found a "real" workaround or if we must wait the next update of gulp-mjml ?

**Comment by iRyusa** [comment]
The fix should be available tomorrow ! Sorry about that

**Comment by SMontenez** [comment]
Hi, same problem here !

Thanks for the fix, is it possible to also remove the "latest" dependency on mjml ? It's quite dangerous for our production app, as updates are made without any warning.
As you take care of both libraries, I think you could handle versions of mjml manually here so users of gulp-mjml are aware of version bumps.

**Comment by iRyusa** [comment]
👍 Agree

**Comment by seanhak** [comment]
Hi, Is the fix out? 
I'm still "stuck" at v 1.1 with above issues. Any temporary fixes advised?

**Comment by iRyusa** [comment]
The fix is here : #12 it should be available today
